url_parser_test.go

```diff
@@ -494,6 +494,18 @@ func TestUnescapeStringTokens(t *testing.T) {
 			expectedFilterTree: nil,
 			expectedOrderBy:    nil,
 		},
+		{
+			url:                "/Product?$orderby=Name,",
```

Comment:
Is there also a problem with extra commas in the middle or in front?
`/Product?$orderby=Name,,Vendor`